### PR TITLE
experimental: Add Desktop Webflow Navbar support

### PR DESCRIPTION
**Pull request by istarkov**
## Description

This 
image

To this
image

### All
- w-nav-link preset colors do not match

### Mobile
Mobile burger is not shown 
- w-nav-button does not  have breakpoint style i.e. display: none->block for mobile
- w-nav-menu  does not  have breakpoint related style i.e display: block -> none  for mobile
- we don't have widget icons like burger i.e. `"data": { "widget": { "type": "icon", "icon": "nav-menu" } }` _having breakpoint issues and that user in any case need to use Sheet on mobile this is ok_

## Steps for reproduction

Copypaste Navbar from webflow 2 webstudio

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

**Review by TrySound — APPROVED** [review]

**Comment by kof** [comment]
- [ ] instance labels need fixing, can we keep wf naming?
image
image

**Comment by kof** [comment]
- [ ] trying to copy navbar 1 from relume

image

seems like I am missing the logo image and some other styles are off - needs investigating, maybe its the css vars or tag styles which is not a bug then

image

File: apps/builder/app/shared/copy-paste/plugin-webflow/plugin-webflow.ts [hunk 1]
```diff
@@ -118,25 +118,53 @@ const parse = (clipboardData: string) => {
...
     toast.info(message);
     console.info(message);
   }
 
   const result = WfData.safeParse(data);
 
   if (result.success) {
+    const unpasedTypes = new Set<string>();
+    // console.log("result.data.payload.nodes", result.data.payload.nodes);
+    for (let i = 0; i !== result.data.payload.nodes.length; ++i) {
```
Comment (kof) [inline apps/builder/app/shared/copy-paste/plugin-webflow/plugin-webflow.ts:144]: can we add a linter for 1 char variables?

File: packages/eslint-config-custom/index.js [hunk 2]
```diff
@@ -38,6 +38,7 @@ module.exports = {
           'ImportDeclaration[source.value="react"] ImportDefaultSpecifier',
       },
     ],
+    "id-length": [2, { exceptions: ["i", "j", "k", "_"], properties: "never" }],
```
Comment (TrySound) [inline packages/eslint-config-custom/index.js:41]: the hell, I want my x and y

File: packages/sdk-components-react-radix/src/theme/tailwind-classes.ts [hunk 3]
```diff
@@ -245,10 +247,12 @@ export const my: ReturnType<typeof marginProperty> = (margin) => {
   return [mt(margin), mb(margin)].flat();
 };
 
+// eslint-disable-next-line id-length
```
Comment (TrySound) [inline packages/sdk-components-react-radix/src/theme/tailwind-classes.ts:250]: meh

File: apps/builder/app/shared/copy-paste/plugin-webflow/plugin-webflow.ts (hunk 1 above)
Comment (TrySound) [reply apps/builder/app/shared/copy-paste/plugin-webflow/plugin-webflow.ts:144]: Better no, it breaks x, y, z axis and p, b, I tags which are perfectly valid

**Pull request merged**